src/main/frontend/pipeline-console-view/pipeline-console/main/NoStageStepsFallback.tsx

```diff
@@ -0,0 +1,63 @@
+import { Suspense, useEffect, useState } from "react";
+
+import {
+  getConsoleBuildOutput,
+  StepInfo,
+  StepLogBufferInfo,
+} from "../../../common/RestClient.tsx";
+import { Result } from "../../../pipeline-graph-view/pipeline-graph/main/PipelineGraphModel.tsx";
+import ConsoleLogStream from "./ConsoleLogStream.tsx";
+
+async function fetchData(): Promise<StepLogBufferInfo> {
+  const consoleBuildOutput = await getConsoleBuildOutput();
+
+  return {
+    lines: consoleBuildOutput?.split("\n") ?? [],
+    startByte: 0,
+    endByte: 0,
+  };
+}
+
+export function NoStageStepsFallback() {
+  const step: StepInfo = {
+    id: "step-1",
+    name: "Step 1",
+    title: "Step 1 Title",
+    state: Result.success,
+    completePercent: 100,
+    type: "STEP",
+    startTimeMillis: 0,
+    totalDurationMillis: 0,
+    stageId: "",
+    pauseDurationMillis: 0,
+  };
+
+  const [logBuffer, setLogBuffer] = useState<StepLogBufferInfo>();
+
+  useEffect(() => {
```

Comment:
I could leave this commit out, its a bit of a PoC really and quite hacky.

<img width="1397" alt="image" src="https://github.com/user-attachments/assets/ba6b6f1b-e710-464c-a0a3-ca5b9979b046" />

It means when there's no flow nodes the build log can be shown instead.